scripts/index.js

```diff
@@ -0,0 +1,74 @@
+function getRandomInt(min, max) {
+	min = Math.floor(min);
+	max = Math.ceil(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+
+let diceOne = {
+	value: 1,
+	roll : function(){
+		this.value = getRandomInt(1, 6);
+		return this.value;
+	}
+};
+let diceTwo = {
+	value: 1,
+	roll : function(){
+		this.value = getRandomInt(1, 6);
+		return this.value;
+	}
+};
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+
+
+let button = document.body.querySelector("#roll-dice");
+
+
+
+const loadDice = event =>{
+	if(event) {
+		event.preventDefault;
+	}
+	
+
+	let containerOne = document.body.querySelector("#container-1");
+	const diceUno = diceOne.roll();
```

Comment:
You don't actually have to store the return of the dice into a variable. You can also just roll the dice and then pass `dice.value` into the `printDice()` function below, since the role method **actually** changes the `value` property on the dice object. Like this:

```javascript
dice.roll();
dice2.roll();

printDice(dice.value, dice2.value);